genai-perf/tests/test_rankings_converter.py

```diff
@@ -0,0 +1,196 @@
+# Copyright 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
If we're going to start making separate test files for each converter, we should see if we can have a subdirectory for the tests for the converters (same for the retriever tests). This can be in a separate PR, if necessary.

Reply:
I think we are going to have a few PRs soon that reorganize the file hierarchy.
I agree but i would prefer to have them be separate.

Reply:
will address this in a separate PR